### ISSUE TITLE
Move release bundling entirely to webpack

### ISSUE THREAD

**Issue by karmacoma** [LiskHQ/lisk#1348]
### Expected behavior

The `grunt release` task uses `webpack` exclusively to prepare release files.

### Actual behavior

The `grunt release` task uses a combination of `grunt-obfuscator` to prepare `app.js` and then `webpack` for preparing `workersController.js` (used by socketcluster). 

### Steps to reproduce

Execute the `grunt release` task.

### Which version(s) does this affect? (Environment, OS, etc...)

Version `1.0.0`.

**Comment by vitaly-t** [comment]
This should be escalated in priority, as `grunt release` doesn't work anymore at all, producing:

```
Warning: Task "obfuscator" failed. Use --force to continue.
```

possibly, due to the presence of ES6 code in many places now.

**Comment by karmacoma** [comment]
We have decided to simplify the release process by removing application bundling entirely. Upon `grunt release` production files will simply be copied in place and archived.

**Comment by diego-G** [comment]
Closed in favour of #1491
